index.html

```diff
@@ -50,13 +50,10 @@
       </a>
 
       <!-- Teaching Timetables -->
-      <div class="card">
-        <div class="card-title">Teaching Timetables</div>
-        <div class="card-grid sub-card">
-          <a class="left" href="https://timetable.hw.ac.uk/WebTimetables/LiveED/login.aspx">Edinburgh</a>
-          <a class="right" href="https://timetable.hw.ac.uk/WebTimetables/LiveDU/login.aspx">Dubai</a>
-        </div>
-      </div>
+      <a class="card" href="https://timetableexplorer.hw.ac.uk/">
+        <p class="card-title">Teaching Timetables</p>
+        <p class="card-text">timetableexplorer.hw.ac.uk</p>
+      </a>
```

Comment:
Nice, looks good!

![image](https://github.com/user-attachments/assets/9193851c-bb57-435a-9a88-292897976dff)

Reply:
Turns out I need to resolve this conversation to let the PR through.